### ISSUE TITLE
Not able to open Android Auto on Renault Zoe Z.E.50

### ISSUE THREAD

**Issue by Snirpo** [Snirpo/aawireless-issues#38]
Hello,

I use AA Wireless with my Renault Zoe Z.E.50 (that is the new generation with Easylink) and use the option of "automatically request focus" in the developer settings.

Sometimes (not every time) I am not able to open the Android Auto "screen" on EasyLink.
I see that the AA Wireless device has booted, my phone has also made the Android-Auto connection, and "Android Auto" is shown at the top of the Easylink Screen in the car (so Android Auto is available) and even the sound-source in Easylink has already connected with Android Auto.

But the display is not starting to show Android Auto. And if I click at that Android Auto button at the top of the Easylink screen it is not able to actually open the Android Auto screen.

If I disconnect and reconnect the AA Wireless device, it works fine.

So the general impression is: a connection is being made, but the actual display (headend) fails.
I use USB-mode "Direct" and have "Passthrough" disabled.

If I can do anything specific to help with debugging this issue, please let me know.

https://cpeb.freshdesk.com/a/tickets/654

**Comment by Snirpo** [comment]
Fix is probably to make the autofocus setting configurable via the app, or set it to on by default.

**Comment by mfonville** [comment]
I am now using AA without the automatic focus toggled. And not experiencing the issue anymore. Surprising is that Easylink still switches most of the times automatically to Android Auto even without this setting

**Comment by mfonville** [comment]
I got the initial issue again even without the autofocus toggle.

I had opened the car to put some stuff in the boot. But not the driver door. This puts power on the USB, so AAWireless is toggled on. And the phone was within reached and connected to AAWireless wifi probably. But because the driver door wasn't opened, Easylink was not started.

I went away from the car. Not sure if WiFi disconnected or not (but my guess would be it did disconnect, because of thick concrete walls).

When I approached the car again and opened the driver door, Easylink booted.
Android Auto button was visible again, and audio was connected to my Android Auto podcast.
But when pressing the Android Auto button, it was again without result. Had to disconnect AAWireless device to reconnect and use Android Auto.
